### ISSUE TITLE
Gangs Peerk Knife Damage

### ISSUE THREAD

**Issue by MasterWindowS** [Headline22/Gangs#50]
Hi, I've been trying for a long time to fix this problem that won't let me activate the gang knife damage. I increase the damage to 8/8 and it does the default damage. Could you help me a bit with that issue?

**Comment by Headline** [comment]
This plugin is no longer supported. You'll have to wait until someone forks this and maintains it themselves or you use another plugin.

**Comment by Bara** [comment]
It is possible that you've a bayonet as knife? This will not detected by the [current code](https://github.com/Headline/Gangs/blob/master/hl_gangs.sp#L541). Classname of the bayonet is `weapon_bayonet`.

**Comment by MasterWindowS** [comment]
No, I don't have the bayonet knife, and if it works the plugin once I fixed it myself, but it was last year and I don't remember how I fixed it. Actually there is a server with that plugin working that I edited (the plugin) and I don't remember what I did to make it work.
